{
  "cve_id": "CVE-2021-43927",
  "timestamp": "2025-01-10 07:49:08",
  "phases": {
    "primary": {
      "completed": true,
      "timestamp": "2025-01-08 10:43:25"
    },
    "secondary": {
      "completed": true,
      "timestamp": "2025-01-09 11:55:03"
    },
    "extraction": {
      "completed": true,
      "timestamp": "2025-01-10 05:22:56"
    }
  },
  "urls": [
    {
      "url": "https://www.synology.com/security/advisory/Synology_SA_22_01",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-09 20:44:49"
    }
  ]
}